.github/workflows/CI.yml

```diff
@@ -13,16 +13,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - "1.0"
-          - "1"
-          - "nightly"
-        os:
-          - ubuntu-latest
-          - macOS-latest
-          - windows-latest
-        arch:
-          - x64
+        version: ["1.10", "1", "nightly"]
```

Comment:
I personally like the one-per-line style better, it's more git-blame-friendly, I don't see the need for this change. If anything, I'd replace the minimum version with "`min`"